dask_glm/regularizers.py

```diff
@@ -9,6 +9,7 @@ class Regularizer(object):
     Defines the set of methods required to create a new regularization object. This includes
     the regularization functions itself and it's gradient, hessian, and proximal operator.
     """
+    _name = '_base'
```

Comment:
I think maybe this should be just `name`. I think @moody-marlin mentioned that users might defining their own regularizers, so they shouldn't need to override private attributes.